osu.Game/Database/DatabaseWorkingBeatmap.cs

```diff
@@ -14,8 +14,8 @@ internal class DatabaseWorkingBeatmap : WorkingBeatmap
     {
         private readonly BeatmapDatabase database;
 
-        public DatabaseWorkingBeatmap(BeatmapDatabase database, BeatmapInfo beatmapInfo, BeatmapSetInfo beatmapSetInfo, bool withStoryboard = false)
-            : base(beatmapInfo, beatmapSetInfo, withStoryboard)
+        public DatabaseWorkingBeatmap(BeatmapDatabase database, BeatmapInfo beatmapInfo, BeatmapSetInfo beatmapSetInfo = null, BeatmapMetadata metadata = null, bool withStoryboard = false)
+            : base(beatmapInfo, beatmapSetInfo, metadata, withStoryboard)

```

Comment:
Same here. This argument also seems unused, was there a specific reason for this being done?